### PR TITLE
Add scenario management to strategic model

### PR DESCRIPTION
**Pull request by NienkeWagenaar**
## Fixes/Addresses/Summary/Motivation:

The purpose of this PR is to add scenario management to the strategic model. The model takes several input files as input, solves the model and will create a separate output file for each scenario.

## Changes proposed in this PR:
- Change to input settings where user can define path to folder, and list input files. We assume the input files will always be in the same folder
- Moved the settings to the top to create a separate section to indicate to user that user input is required
- Added for loop to solve the model for each input file (/scenario)
- Result file name is updated to "results_opt_strategic" + name of input file, to clearly indicate which result file corresponds to which input file. 

TODO:

- [ ] Restructure operational model run file
- [ ] Apply scenario management for operational model

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: pareto/strategic_water_management/run_strategic_model.py [hunk 1]
```diff
@@ -96,34 +96,31 @@
...
-        "pipeline_cost": PipelineCost.distance_based,
-        "pipeline_capacity": PipelineCapacity.input,
-        "node_capacity": IncludeNodeCapacity.true,
-    },
-)
+    objective: [Objectives.cost, Objectives.reuse]
+    pipeline_cost: [PipelineCost.distance_based, PipelineCost.capacity_based]
+    pipeline_capacity: [PipelineCapacity.input, PipelineCapacity.calculated]
+    node_capacity: [IncludeNodeCapacity.True, IncludeNodeCapacity.False]"""
+objective = Objectives.cost
```
Comment (RuudWag) [inline pareto/strategic_water_management/run_strategic_model.py:118]: You can create the default object over here and pass it as an argument to the create_model function

Comment (NienkeWagenaar) [reply pareto/strategic_water_management/run_strategic_model.py:118]: Done.

**Review by RuudWag — DISMISSED** [review]

**Comment by melody-shellman** [comment]
Per discussion 10/11/2022:
The UI accomplishes much of what this PR was set to accomplish. This PR could be a "low hanging fruit" to add a new feature for users, but it is LOW priority. To revisit in 2-4 weeks.

**Comment by melody-shellman** [comment]
Desired results already largely implemented in UI. Additionally, this feature is not included in 2023 Milestones, so closing.

**Pull request closed without merging**